src/labels/CustomLabels.labels

```diff
@@ -34,36 +42,44 @@
     </labels>
     <labels>
         <fullName>PermissionCreateException</fullName>
-        <categories>CampaignTools,Exceptions</categories>
+        <categories>Exceptions</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>PermissionCreateException</shortDescription>
         <value>You don&apos;&apos;t have the permissions required to create {0}. Please contact the system administrator.</value>
     </labels>
     <labels>
         <fullName>PermissionDeleteException</fullName>
-        <categories>CampaignTools,Exceptions</categories>
+        <categories>Exceptions</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>PermissionDeleteException</shortDescription>
         <value>You don&apos;&apos;t have the permissions required to delete {0}. Please contact the system administrator.</value>
     </labels>
     <labels>
         <fullName>PermissionReadException</fullName>
-        <categories>CampaignTools,Exceptions</categories>
+        <categories>Exceptions</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>PermissionReadException</shortDescription>
         <value>You don&apos;&apos;t have the permissions required to view {0}. Please contact the system administrator.</value>
     </labels>
     <labels>
         <fullName>PermissionUpdateException</fullName>
-        <categories>CampaignTools,Exceptions</categories>
+        <categories>Exceptions</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>PermissionUpdateException</shortDescription>
         <value>You don&apos;&apos;t have the permissions required to update {0}. Please contact the system administrator.</value>
     </labels>
+    <labels>
+        <fullName>TelemetryException</fullName>
+        <categories>Exceptions</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error capturing metrics.</shortDescription>
+        <value>An unknown error occurred when attempting to capture usage metrics.</value>
```

Comment:
@katherinemitchell - similar to the previous error, this will only show up in the off chance the job fails.

Reply:
line 81 - "An error occurred when attempting to capture usage metrics."